### PR TITLE
feat/simpconsent: 간편서명동의 설정 프론트 구현, 서명이미지 S3 연동

### PR DESCRIPTION
**Pull request by EunaAhn**
## TYPE
- [X] ✅ Feat
- [X] 🔧 Fix
- [ ] ✨ Style
- [ ] 📖 Test
- [ ] 📝 Refactor
- [ ] 📈 Perf
- [ ] 📂 Docs
- [ ] 🅱 Build
- [ ] 🔄 CI

## DESCRIPTION
- 간편동의설정 프론트 구현
- 서명 이미지 S3 연동

## ISSUE
- [KAN-155](https://hyosung-kosa-team3.atlassian.net/browse/KAN-155?atlOrigin=eyJpIjoiZTlhZjM5ZTljNDZhNDY3Njk0ZjJiMDEzYTZiMDk4YzkiLCJwIjoiaiJ9)
- [KAN-122](https://hyosung-kosa-team3.atlassian.net/browse/KAN-122?atlOrigin=eyJpIjoiMDc0MTliYjc1YmQ3NDgyNGFkODdiM2FiNTI4OWY2MWYiLCJwIjoiaiJ9)
- [KAN-168](https://hyosung-kosa-team3.atlassian.net/browse/KAN-168?atlOrigin=eyJpIjoiOTg3NjUzNmNiNmEyNDI2ZDllN2IzZTZmZjE2MzRkNzAiLCJwIjoiaiJ9)

[KAN-155]: https://hyosung-kosa-team3.atlassian.net/browse/KAN-155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[KAN-122]: https://hyosung-kosa-team3.atlassian.net/browse/KAN-122?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[KAN-168]: https://hyosung-kosa-team3.atlassian.net/browse/KAN-168?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: client/src/components/vendor/modal/SimpConsentQrUrlModal.jsx [hunk 1]
```diff
@@ -1,14 +1,100 @@
...
+  useEffect(() => {
+    if (qrRef.current) {
+      const canvas = qrRef.current.querySelector('canvas');
+      QRCode.toCanvas(canvas, url, { width: 200 }, error => {
+        if (error) console.error('Error generating QR code', error);
+      });
+    }
+  }, [url]);
+
+  const handleCopyUrl = () => {
```
Comment (rlatkd) [inline client/src/components/vendor/modal/SimpConsentQrUrlModal.jsx:20]: 커스텀 모달(ex. sweatAlert 등)을 구현하면 좋을 것 같습니다.

File: client/src/components/member/simpConsent/PaymentCMS.jsx [hunk 2]
```diff
@@ -3,6 +3,21 @@ import Input from '@/components/common/Input';
 import SelectField from '@/components/common/SelectField';
 import { useUserDataStore } from '@/stores/useUserDataStore';
 
+const bankOptions = [
```
Comment (hyo-nu) [inline client/src/components/member/simpConsent/PaymentCMS.jsx:6]: 컴포넌트 밖으로 뺀거 좋은것 같습니다!

**Review by rlatkd — APPROVED** [review]
good

File: server/src/main/java/kr/or/kosa/cmsplusmain/domain/s3/FileUploadController.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+            amazonS3Client.putObject(bucket, fileName, file.getInputStream(), metadata);
+            return ResponseEntity.ok(fileUrl);
+        } catch (IOException e) {
+            e.printStackTrace();
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body("파일 업로드 중 오류 발생: " + e.getMessage());
+        } catch (Exception e) {
+            e.printStackTrace();
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body("예상치 못한 오류 발생: " + e.getMessage());
+        }
```
Comment (pogihae) [inline server/src/main/java/kr/or/kosa/cmsplusmain/domain/s3/FileUploadController.java:51]: service 레이어로 분리하면 좋을 것 같아용

```.java
String fileName = UUID.randomUUID() + ....
// fileName을 회원과 계약을 특정 지을 수 있는 값으로 넣어두면 좋을 것 같아요
```

**Review by pogihae — APPROVED** [review]

File: client/src/components/vendor/modal/SimpConsentQrUrlModal.jsx (hunk 1 above)
Comment (hyo-nu) [inline client/src/components/vendor/modal/SimpConsentQrUrlModal.jsx:20]: 모바일 용 Alert도 컴포넌트화 해놓겠습니다!

**Review by hyo-nu — APPROVED** [review]

File: client/src/components/vendor/modal/SimpConsentQrUrlModal.jsx (hunk 1 above)
Comment (EunaAhn) [reply client/src/components/vendor/modal/SimpConsentQrUrlModal.jsx:20]: 넵!

File: server/src/main/java/kr/or/kosa/cmsplusmain/domain/s3/FileUploadController.java (hunk 3 above)
Comment (EunaAhn) [reply server/src/main/java/kr/or/kosa/cmsplusmain/domain/s3/FileUploadController.java:51]: 수정하겠습니다!

**Pull request merged**